CoroutineTemplate/app/src/main/res/layout/fragment_home.xml

```diff
@@ -12,7 +13,8 @@
         android:text="Next"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="HardcodedText" />
```

Comment:
Why we don't add string in `strings.xml` instead of ignoring `HardcodedText` 🙏

Reply:
I'll add it to `strings.xml` 🚀

Reply:
I don't think we should try to add those example detail stuff into config like strings or dimension. We should try to keep our Template project neat, clean, and flexible by the `setup` script. 🤔 

FYI: this is the same issue we're discussing when building Flutter Template https://github.com/nimblehq/flutter_templates/issues/58 🤓

Reply:
We're also planning to restructure the folders in this repository, it would be nice if both flutter and android templates could agree on the same implementation. Most likely this discussion will not be concluded this month, I'll need some more time to think about it too.

@Wadeewee Would it be OK to keep the changes for now? Once we've agreed on how to restructure the project, we can have another look into separating the strings 🙏

Reply:
> Would it be OK to keep the changes for now? Once we've agreed on how to restructure the project, we can have another look into separating the strings

@Tuubz Sound good to me 🚀